src/main/java/appeng/core/api/definitions/ApiBlocks.java

```diff
@@ -222,8 +222,7 @@ public ApiBlocks( FeatureFactory registry, PartModels partModels )
 		this.chiseledQuartzBlock = deco.block( "chiseled_quartz_block", BlockChiseledQuartz::new ).build();
 		this.quartzGlass = deco.block( "quartz_glass", BlockQuartzGlass::new )
 				.useCustomItemModel()
-				.rendering( new BlockRenderingCustomizer()
-				{
+				.rendering( new BlockRenderingCustomizer(){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)